### DISCUSSION TITLE
Howto reference static HTML doc in public using the markdown link syntax?

### DISCUSSION THREAD

**Discussion by snowyu** [vuepress/core#1248] [Q&A]
HTML files placed in `.vuepress/public` are only sporadically available. Trying to navigate to them from a link elsewhere on the site results in a 404. However, reloading the page finds the content.

```markdown
* [external html page in public dir](/dir_in_public/index.html)
```

**Comment by Mister-Hope** [comment]
Use `<a>` tag directly

**Reply by snowyu** [reply]
1. `<a href="/dir_in_public/index.html">`  still be translated to `/dir_in_public%2Findex.html`. And then `404` too.
2. It's an HTML syntax.
